mmf/models/movie_mcan.py

```diff
@@ -226,7 +226,13 @@ def process_feature_embedding(
             feature = sample_list.image
 
             feature_encoder = getattr(self, attr + "_feature_encoders")
-            encoded_feature = feature_encoder(feature, text_embedding_vec)
```

Comment:
Just for my knowledge, does removing the `text_embedding_vec` change anything. I checked that the default config is only default encoder. Maybe I am misunderstanding something here.

Reply:
`text_embedding_vec` is not required for image feature encoders. This was old code and was not verified properly in MMF.